packages/domain-picker/src/index.tsx

```diff
@@ -0,0 +1,5 @@
+/**
+ * Internal dependencies
+ */
+import DomainPicker from './domain-picker';
+export default DomainPicker;
```

Comment:
I think this should work:

```suggestion
export { default } from './domain-picker';
```

Alternatively, we can avoid this file entirely and name the domain-picker file `index`.

Reply:
Separate export index-file would be important for making easy-to-treeshake bundles only if we'd want to export multiple things, but if we have just one thing, we can rename domain-picker file to index.

Reply:
> Alternatively, we can avoid this file entirely and name the domain-picker file index.

Style lint wants the folder name to be the className prefix, e.g. if your folder name is `src`, your classname should start with `src__*` prefix, which is weird. That's why it was moved to `domain-picker` folder, so we can retain the `domain-picker__*` classname prefix.

> Separate export index-file would be important for making easy-to-treeshake bundles only if we'd want to export multiple things.

That's the idea!